Applications/MainApplication/Gui/MainWindow.cpp

```diff
@@ -229,16 +229,16 @@ void MainWindow::onUpdateFramestats( const std::vector<FrameTimerData>& stats )
 
     for ( uint i = 0; i < stats.size(); ++i )
     {
-        sumEvents += Core::Timer::getIntervalMicro( stats[i].eventsStart, stats[i].eventsEnd );
-        sumRender += Core::Timer::getIntervalMicro( stats[i].renderData.renderStart,
+        sumEvents += Core::Utils::getIntervalMicro( stats[i].eventsStart, stats[i].eventsEnd );

```

Comment:
Timer deserve a class (even if everything is static) I will add a comment on this later on, but here we should have
Core::Utils::Timer::getIntervalMicro ....